charts/argo-rollouts/values.yaml

```
@@ -15,6 +15,10 @@ nameOverride:
 # -- String to fully override "argo-rollouts.fullname" template
 fullnameOverride:
 
+# -- Override the namespace
+# @default -- `.Release.Namespace`
+namespaceOverride: argo-rollouts
```

Maybe it's supposed to be blank since we cannot be sure if the users name the namespace as `argo-rollouts` .
e.g. ) https://github.com/argoproj/argo-helm/blob/main/charts/argo-cd/values.yaml#L11
```suggestion
namespaceOverride: ""
```